docs/docs/getting-started/troubleshooting.md

```diff
@@ -4,35 +4,20 @@
 
 ### hydra-node
 
-- Hydra node crashes after a fork
-
-    + It ocurs during a fork and expects the operator to restart its hydra-node.
-
-    + Reference: [issue #560](https://github.com/input-output-hk/hydra/issues/560) Hydra node crashed after a fork.
-    
-    + Workaround: Restarting your node should be enough to come back to live.
-
```

Comment:
I would not remove this item as it looks like this removal is unrelated to this branch purpose

Reply:
It is related since this info is outdated now. We merged yesterday what we hope is a fix for this so unless we see this happening again we consider it fixed.